### PR TITLE
Feature integration with flask

### PR DESCRIPTION
**Pull request by jassalss**
This is my PR for integration. I would say the first phase of integration is done. Integration for profile_settings screen is complete. Integration for the confirm screen is 50% complete. I will connect the availability screen to flask when it will finish.
All screens are connected now.

### CODE REVIEW

File: client/src/component/ProfileWidget.js [hunk 1]
```diff
@@ -2,133 +2,163 @@ import React from "react";
...
+                      defaultValue: "john-doe",
+                    }}
+                    onChange={handleChangeInUrl}
+                  />
+                </FormControl>
+              </Box>
+            </Grid>
+          </Grid>
+          <Grid container alignItems="center">
+            <Grid item sm>
```
Comment (sina-jamshidi) [inline client/src/component/ProfileWidget.js]: probably looking for this to be `sm={4}`

File: server/api/check_unique_url.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def check_url_is_unique(user_id):
             unique_url = User.find_url(given_url)
             if unique_url:
                 output["error"] = "Unavailable"
+                status = 200
```
Comment (sina-jamshidi) [inline server/api/check_unique_url.py:26]: status 400?

**Review by sina-jamshidi — APPROVED** [review]
looks good! just minor things

File: client/src/component/ProfileWidget.js (hunk 1 above)
Comment (jassalss) [reply client/src/component/ProfileWidget.js]: fixed

File: server/api/check_unique_url.py (hunk 2 above)
Comment (jassalss) [reply server/api/check_unique_url.py:26]: Reason for 200 is that, API call is successful in both cases, whether we find unique url or not.

**Pull request merged**